### DISCUSSION TITLE
支持接入本地模型

### DISCUSSION THREAD

**Discussion by fecet** [immersive-translate/immersive-translate#1088] [Ideas]
### 新功能 | New Feature

能否提供需要翻译的文本api, 允许用户使用本地或者自己的模型部署

### 为什么您想要添加这些功能 | Why you want to add these features

1.
2.
3.

### 提议 | Proposal

提供api返回需要翻译的文本列表, 并允许我post翻译好的文本

**Comment by FattySpring** [comment]
https://immersivetranslate.com/docs/services/custom/ 可以看下这个文档
